test/src/click.spec.ts

```diff
@@ -164,7 +164,10 @@ describe('Page.click', function () {
     await page.goto(server.PREFIX + '/offscreenbuttons.html');
     const messages: any[] = [];
     page.on('console', msg => {
-      return messages.push(msg.text());
+      if (msg.type() === 'log') {
```

Comment:
why was this change needed?

Reply:
Yes, that's because in Firefox when I call `await this._client.send('DOM.scrollIntoViewIfNeeded')` the error appears in the console and the test case should observe only the log messages.